assets/src/dashboard/app/views/myStories/karma/myStories.karma.js

```diff
@@ -141,7 +141,9 @@ describe('Grid view', () => {
 
     utils = within(firstStory);
 
-    expect(utils.getByText(/Copy/)).toBeTruthy();
+    const copiedStory = utils.queryAllByText(/Copy/)[0];
+
+    expect(copiedStory.text.includes('(Copy)')).toBeTruthy();
```

Comment:
Had to adjust these tests because now `Copy` appears in the firstStory node more than once.